### DISCUSSION TITLE
How to merge two GitHub accounts?

### DISCUSSION THREAD

**Discussion by lod93** [community/community#132892] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I read the terms to find i can only one GitHub account , so how do I request to merge them into one account that have all the history and repos, also activity and badges?

**Comment by davevad93** [comment]
Hi @lod93 ,

Take a look at the GitHub docs about [Merging multiple personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts), just bear in mind a couple of things:

1. You can't transfer achievements between accounts.
2. Issues, pull requests, and discussions won't be attributed to the new account.
3. Commits authored with a noreply email address can't be transferred from one account to another.
